xml/System.Formats.Nrbf/NrbfDecoder.xml

```diff
@@ -55,12 +55,21 @@
         <Parameter Name="leaveOpen" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="payload">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="leaveOpen">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="payload">The NRBF payload.</param>
+        <param name="options">Options to control behavior during parsing.</param>
+        <param name="leaveOpen">
+          <see langword="true" /> to leave <paramref name="payload" /> payload open after the reading is finished; otherwise, <see langword="false" />.</param>
+        <summary>Decodes the provided NRBF payload.</summary>
+        <returns>A <see cref="T:System.Formats.Nrbf.SerializationRecord" /> that represents the root object.
+ It can be either <see cref="T:System.Formats.Nrbf.PrimitiveTypeRecord`1" />, a <see cref="T:System.Formats.Nrbf.ClassRecord" /> or an <see cref="T:System.Formats.Nrbf.ArrayRecord" />.</returns>
```

Comment:
```suggestion
 It can be either <see cref="T:System.Formats.Nrbf.PrimitiveTypeRecord`1" />, a <see cref="T:System.Formats.Nrbf.ClassRecord" />, or an <see cref="T:System.Formats.Nrbf.ArrayRecord" />.</returns>
```